src/tools/fuzzing/fuzzing.cpp

```diff
@@ -3041,6 +3041,12 @@ HeapType TranslateToFuzzReader::getSubType(HeapType type) {
 }
 
 Rtt TranslateToFuzzReader::getSubType(Rtt rtt) {
+  if (getTypeSystem() == TypeSystem::Nominal) {
```

Comment:
We should include `TypeSystem::Isorecursive` in this condition as well, since it will have the same problem.

Reply:
done.